src/examples/copy/main.rs

```diff
@@ -1,16 +1,11 @@
 /* --- BEGIN Variable Definitions ---
-Owner x; Owner y
+Owner x; Owner y; Owner k;
```

Comment:
this should be a new example, not a change to the copy example?

Reply:
yeah, that seems to be when i want to test some basic functionality in original rustviz. Will unroll that change